libqfieldsync/project_checker.py

```diff
@@ -289,6 +294,36 @@ def check_project_is_dirty(self) -> Optional[FeedbackResult]:
         else:
             return None
 
+    def check_for_conflicting_base_filenames(self) -> Optional[FeedbackResult]:
+        conflicting_files: List[Path] = []
+
+        project_file_path = Path(self.project.fileName())
+        project_home_path: Path = project_file_path.parent
+        project_base_name: str = project_file_path.stem
```

Comment:
```suggestion
        project_home_path = project_file_path.parent
        project_base_name = project_file_path.stem
```

No need for these, they are derived from the typing of the `Path` object.

Reply:
Completely agree!